gluten-data/src/main/java/io/glutenproject/init/NativeBackendInitializer.java

```diff
@@ -36,7 +36,7 @@ public static void initializeBackend(scala.collection.Map<String, String> conf)
       BackendsApiManager.getSettings().resolveNativeConf(nativeConfMap);
       initialize(JniUtils.toNativeConf(nativeConfMap));
     } catch (Exception e) {
-      LOG.error("Error calling InitializerJniWrapper.initialize(...)", e);
+      LOG.error("Error calling NativeBackendInitializer.initialize(...)", e);
```

Comment:
Thanks for your fix!
Previously, we use `InitializerJniWrapper.initialize` to do the initialization. Now it becomes `initialize(...)`. So it may be better to change the error message to: "Failed to call native backend's initialize method" or something else.

Reply:
updated